recipe/meta.yaml

```
@@ -10,33 +10,47 @@ source:
   sha256: fba21101fd9e29d5e3b72823261aaae06b9350686f0d2067786d64dce73eb3f6
 
 build:
+  number: 0
+  # noarch is used. python issue on osx-64:
+  # - https://github.com/python/cpython/issues/91310
+  # - https://github.com/python/cpython/issues/90849
+  # Workaround with ARCHFLAGS="-arch x86_64" not worked
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 1
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  #[py<36]
 
 requirements:
   host:
-    - flit-core >=3.2,<4
     - pip
-    - python {{ python_min }}
+    - python
+    - wheel
+    - setuptools
+    - flit-core >=3.2,<4
   run:
-    - python >={{ python_min }}
+    - python
```

```suggestion
    - python >=3.9
```
It's needed for noarch python